docs/howto/DEBUG_IDP.md

```diff
@@ -0,0 +1,17 @@
+# Debug tokens from IDP
+
+## When/why?
+
+When it's not really well documented on IDP's side which specific fields names should be used in plugin's attribute mapping.
```

Comment:
Most common Idps do document this somewhere.
Perhaps it would be worth creating a table of common values for well known Idps